packages/gdl-frontend/style/__tests__/media.test.js

```diff
@@ -6,7 +6,8 @@
  * See LICENSE
  */
 
-import { css } from 'react-emotion';
+import { css } from '@emotion/core';
+// $FlowFixMe getStyles does not exist, but since this test is skipped removes error for now
 import { getStyles } from 'jest-emotion';
```

Comment:
Suppressed flow error her foreløpig. Testen skippes, men på sikt ønsker vi en fix hvor det skal finnes en funksjon som kan trekke ut css i `templateLiteral` for å opprette snapshot.